dashboard-ui/src/pages/home.tsx

```diff
@@ -400,8 +407,11 @@ const DisplayItems = ({
     });
   }
 
+  // not showing all the rows at once when user selects all workloads filter
+  const allWorkloadView = useMemo(() => workloadFilter === undefined, [workloadFilter]);
+
   // handle show some-or-all
-  const [showAll, setShowAll] = useState(false);
+  const [showAll, setShowAll] = useState(!allWorkloadView);
```

Comment:
Do we need to track the state of `showAll`? Why not just use the value of `workloadFilter` itself?

Reply:
That sounds good too, will try this